synapse/rest/client/v2_alpha/account.py

```diff
@@ -261,10 +261,7 @@ async def on_POST(self, request):
                 if "medium" not in threepid or "address" not in threepid:
                     raise SynapseError(500, "Malformed threepid")
                 if threepid["medium"] == "email":
-                    # For emails, transform the address to lowercase.
-                    # We store all email addreses as lowercase in the DB.
-                    # (See add_threepid in synapse/handlers/auth.py)
```

Comment:
again, most of this comment is correct: could you update it rather than remove it?